modules/data-table/test/spec.coffee

```diff
@@ -1558,6 +1560,114 @@ describe 'data-table', ->
           ])
           done()
 
+
+      describe 'advanced search', ->
```

Comment:
New tests for the advanced search inside the feed - Tests are still in progress for the _actual_ advanced search, this just deals with the data and making sure it is filtered using Distributed Normal Form